Star Wars Contacts.xcodeproj/project.pbxproj

```diff
@@ -74,8 +74,8 @@
 			isa = PBXFrameworksBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
-				273F1EE522A87B4700237A82 /* Alamofire in Frameworks */,
```

Comment:
The project wouldn't compile as is.  I tried deleting Package.resolved and restarting but in the end I deleted the package references and added the same package url references